quickstart-server.yml

```diff
@@ -34,6 +34,8 @@ services:
       - IPFS_ENDPOINT
       - IPFS_CLIENT_ENDPOINT
       - IPFS_MAX_FILE_SIZE
+      - AUTH_ORY_KRATOS_ISSUER
```

Comment:
Good while modifying this file to also remove the AAD / DEMO AUTH env vars

Reply:
Fixed in f47154b44e7560df4e8aae338ccfb66e68d84677